plugins/thunderbolt/fu-thunderbolt-controller.c

```diff
@@ -288,6 +288,7 @@ fu_thunderbolt_controller_setup(FuDevice *device, GError **error)
 
 		} else {
 			device_id = g_strdup("TBT-fixed");
+			fu_device_add_parent_guid(device, "cpu");
```

Comment:
i know this is true electrically, but is this what the user is going to think?

Reply:
Nothing really - you only see it in `--show-all-devices` output.  Another argument can be to just not populate the device.  It has nothing valuable in fwupd.

Reply:
> Nothing really - you only see it in --show-all-devices output

Ah, fair point. Got it.